<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CourierWindowClass</class>
 <widget class="QMainWindow" name="CourierWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>743</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CourierWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="zoneLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Zone</string>
    </property>
   </widget>
   <widget class="QListWidget" name="undeliveredPackagesList">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="streetsComboBox">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>60</y>
      <width>68</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="filterPushButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>100</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>filter streets</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deliverButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>260</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Deliver</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>743</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="CourierWindow.qrc"/>
 </resources>
 <connections/>
</ui>
